### PR TITLE
Validate $HACHI

### PR DESCRIPTION
**Pull request by HachikoSolana**
# Validate [$HACHI](https://solscan.io/token/HAchgdhsPRicRTEKZkNH1tcoaAs8gJk1rrJzjA7s5KdN)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://x.com/hachiko_loyal/status/1808852249969348748?s=46
- Coingecko/ CMC URL (If available): https://coinmarketcap.com/currencies/hachiko-run

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Comment by Sorino001** [comment]
TO THE MOON ✈️✈️

**Comment by nizamm** [comment]
$hachiko is legit project will 800+ community

**Comment by Mink2k4** [comment]
Moonnnnn

**Review by nizamm — COMMENTED** [review]
all information provided is accurate

**Comment by Sorino001** [comment]
IMG_5062

**Comment by Skenderbegdardania** [comment]
LFG ! 💥🚀

**Comment by Pkabutu** [comment]
Hachiko will surprise many soon.Best Dev in Solana in here guys

**Comment by criptoeeseb** [comment]
THIS ENTRY LEVEL IS EVERY CHADS DREAM !! CMC PENDING, APPROACHING 1000 MEMBERS IN THE TELEGRAM!!

**Comment by Oc995** [comment]
Great Dev and project

**Review by Pkabutu — COMMENTED** [review]
Great team behind Hachiko.Check it out

**Comment by Antony768** [comment]
Best coin!!! 
great team and doing a lot of good work🔥

File: validated-tokens.csv
Comment (Dota15) [inline validated-tokens.csv:1]: Hachiko is the best #4908

**Comment by nizamm** [comment]
$haciko deserve to be on strict list as secure trading on phantom

**Comment by Skenderbegdardania** [comment]
Moon mission!

File: validated-tokens.csv
Comment (nizamm) [reply validated-tokens.csv:1]: links are all accurate

**Review by nizamm — COMMENTED** [review]
can approve below commit

**Review by Huhop — COMMENTED** [review]
Appears correct

**Comment by nizamm** [comment]
pull request is valid

File: validated-tokens.csv
Comment (JoFo44) [reply validated-tokens.csv:1]: All good

**Review by JoFo44 — COMMENTED** [review]
$Hachi is the real deal

**Review by JJ-cryptoo — COMMENTED** [review]
Supported

**Comment by Dannyboy241** [comment]
This is a legitimate team and the info they provided is accurate I am the wenpad.io developer we have been recently marked safe by blowfish I vouch for this team

**Review by Manttorne — COMMENTED** [review]
Everything seems ready to fly to me!

**Comment by Jymfletzfur** [comment]
image

token does not meet all required metrics

**Pull request closed without merging**